Documentation.md

```diff
@@ -164,590 +174,606 @@ and made available to your JS client code.
 
-### `agent.getChannelConcurrency`
-```
-if (agent.getChannelConcurrency("VOICE")) { ... }
-```
-OR
-```
-const concurrencyMap = agent.getChannelConcurrency();
+### `agent.getChannelConcurrency()`
+```js
+var concurrencyMap = agent.getChannelConcurrency(); // e.g. { VOICE: 0, CHAT: 2 }
+
+// OR
+
+if (agent.getChannelConcurrency("CHAT") > 1) { /* ... */ }
 ```
-Gets either a boolean represented by a 1 or 0, or a map of channel type to 1 or 0. 1 represents an enabled channel. 0 represents a disabled channel.
-* `channel`: A string ENUM representing the channel whose value to return. 
-The ENUM options are currently `"VOICE"` and `"CHAT"`
+Gets either a number or a map of `ChannelType`-to-number indicating how many concurrent contacts can an agent have on a given channel. 0 represents a disabled channel. The optional `channel` argument must be a value of the enum `ChannelType`.
 
 ### `agent.getName()`
-```
+```js
 var name = agent.getName();
 ```
-Gets the agent's user friendly display name from the `AgentConfiguration` object for the agent.
+Gets the agent's user friendly display name.
 
 ### `agent.getExtension()`
-```
+```js
 var extension = agent.getExtension();
 ```
-Gets the agent's phone number from the `AgentConfiguration` object for the agent.  This is the phone
-number that is dialed by Amazon Connect to connect calls to the agent for incoming and outgoing calls if
-softphone is not enabled.
+Gets the agent's phone number. This is the phone number that is dialed by Amazon Connect to connect calls to the agent for incoming and outgoing calls
+if softphone is not enabled.
 
 ### `agent.getDialableCountries`
-```
-if (agent.getDialableCountries().includes(COUNTRY_CODE)){ ... }
+```js
+var countries = agent.getDialableCountries(); // e.g. ["us", "ca"]
 ```
 Returns a list of eligible countries to be dialed / deskphone redirected.
 
 ### `agent.isSoftphoneEnabled()`
+```js
+if (agent.isSoftphoneEnabled()) { /* ... */ }
 ```
-if (agent.isSoftphoneEnabled()) { ... }
-```
-Determine if softphone is enabled for the agent.
+Indicates whether the agent's phone calls should route to the agent's browser-based softphone or the
+telephone number configured as the agent's extension.
 
 ### `agent.setConfiguration()`
-```
+```js
 var config = agent.getConfiguration();
-config.extension = "+12061231234";
+config.extension = "+18005550100";
 config.softphoneEnabled = false;
 agent.setConfiguration(config {
-   success: function() { ... },
-   failure: function() { ... }
+   success: function() { /* ... */ },
+   failure: function(err) { /* ... */ }
 });
 ```
-Updates the agents configuration with the given `AgentConfiguration` object.  The phone number specified must be in E.164 format or the update fails.
+Updates the agent's configuration with the given `AgentConfiguration` object. The phone number specified must be in E.164 format or the update fails.
+Only the `extension` and `softphoneEnabled` values can be updated.
```

Comment:
Tried to update every field of the configuration, and realized only these two values can be updated.